src/qibocal/web/report.py

```diff
@@ -26,3 +27,24 @@ def create_report(path, actions=None):
 
     with open(os.path.join(path, "index.html"), "w") as file:
         file.write(html)
+
+
+def create_autocalibration_report(path, history):
+    """Creates an HTML report for the data in the given path."""
+
+    filepath = pathlib.Path(__file__)
+    with open(os.path.join(filepath.with_name("static"), "styles.css")) as file:
+        css_styles = f"<style>\n{file.read()}\n</style>"
+
+    report = AutoCalibrationReportBuilder(path, history)
+    env = Environment(loader=FileSystemLoader(filepath.with_name("templates")))
```

Comment:
```suggestion
	TEMPLATES = WEB_DIR / "templates"
    env = Environment(loader=FileSystemLoader(TEMPLATES))
```
At this point you should know what to do ;)